src/workbookManager.ts

```diff
@@ -98,10 +99,10 @@ export class WorkbookManager {
 
         const tableXmlString = await zip.file(tableXmlPath)?.async("text");
         if (tableXmlString === undefined) {
-            throw new Error("Table were not found in template");
+            throw new Error("Table was not found in template");
         }
-        const newPivotTable = await this.updatePivotTablesInitialData(tableXmlString, tableData);
-        zip.file(tableXmlPath, newPivotTable);
+        const newTable = await this.updatePivotTablesInitialData(tableXmlString, tableData);
```

Comment:
Seems from our talk this updates a table, and not the pivot table definition - lets rename